hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/TokenFeature.java

```diff
@@ -731,6 +733,41 @@ public void verifyMirrorNftTransfer(AccountClient.AccountNameEnum accountName) {
         verifyNftTransactions(tokenId, serialNumber);
     }
 
+    @Then("I airdrop {int} tokens to {account}")
+    public void airdropTokens(int amount, AccountNameEnum accountName) {
+        var receiver = accountClient.getAccount(accountName);
+        var sender = accountClient.getSdkClient().getExpandedOperatorAccountId();
+        networkTransactionResponse = airdropFungibleTokens(tokenId, amount, sender, receiver);
+    }
+
+    @Then("I verify {string} airdrop of {int} tokens to {account} from {account}")
```

Comment:
Add retry logic so it'll auto retry in case mirrornode data isn't ready.
```suggestion
    @RetryAsserts
    @Then("I verify {string} airdrop of {int} tokens to {account} from {account}")
```

Reply:
done